spec/ics-2-consensus-verification/VERIFIER.md

```diff
@@ -0,0 +1,75 @@
+# Verifier
+
+Full nodes are procedures who run the all `[Message]` according to the
+`Consensus`. The lightclient `Verifier` does not runs all `[Message]`, but
+it uses the `Consensus`'s security assumption to fast-forward the verification
+process. The `Verifier` MUST work identical with a full node while the
+security assumptions of `Consensus` are preserved. Which means, if and only
+if a full node accepts the new `Header` given a `ConsensusState` and
+`[Message]`, then the `Verifier` also accepts it.
+
+## Definitions
```

Comment:
Wait I'm confused there is a definition section above. Why are there two definitions sections in the docs?

Reply:
Oh they are different files, the verifier one is specifically about lightclient verifier internal, where readme is the interface part for the verifier.